src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  Books: [],
+};
+
+const booksSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook(state, action) {
+      state.push(action.payload);
+    },
+    removeBook(state, action) {
+      state.filter((book) => book.id !== action.payload);
+    },
+  },
+});
+
+export const { addBook, removeBook } = booksSlice.actions;
+export default createSlice.reducers;
```

Comment:
- [ ] I must commend you as you have tried 👍🏽, but then, your app can neither add or remove book because you are exporting `createSlice` instead of `bookSlice`. I think its an oversight, so I will recommend you fix it 👍🏽